music_assistant/server/providers/deezer/__init__.py

```diff
@@ -473,7 +474,7 @@ async def get_audio_stream(
             headers["Range"] = f"bytes={skip_bytes}-"
 
         buffer = bytearray()
-        streamdetails.data["start_ts"] = datetime.datetime.utcnow().timestamp()
+        streamdetails.data["start_ts"] = datetime.datetime.now().timestamp()
```

Comment:
this will break soon, better use our utcnow helper from utils.datetime

Reply:
Why would this break soon - I thought `utcnow` was deprecated only? 
However it's cleaner to use UTC timestamps anyways, I changed it 👍

Reply:
datetime.now() without timezone is deprecated and will be remove din python 13